capsules/classifier_safety_gear_openvino/backend.py

```diff
@@ -0,0 +1,85 @@
+from typing import Dict, Union
+
+import numpy as np
+
+from vcap import (
+    DETECTION_NODE_TYPE,
+    OPTION_TYPE,
+    BaseStreamState,
+)
+
+from vcap_utils import (
+    BaseOpenVINOBackend,
+    linear_assignment,
+    iou_cost_matrix,
+)
+
+from .config import (
+    safety_hat,
+    safety_vest,
+    safety_gears,
+    attributes,
+)
+
+from . import config
+
+Num = Union[int, float]
```

Comment:
This isn't required. Just use `float`: https://www.python.org/dev/peps/pep-0484/#the-numeric-tower

Reply:
Actually, is this even used?

Reply:
It was used, but now it's not. I was wondering why PythonCharm not giving me a hint for that. I'm going to remove it.